src/gtest/test_forkmanager.cpp

```diff
@@ -286,3 +287,20 @@ TEST(ForkManager, SidechainForkMainnet) {
 	EXPECT_EQ(ForkManager::getInstance().getNewBlockVersion(1047624), BLOCK_VERSION_SC_SUPPORT);
 	EXPECT_EQ(ForkManager::getInstance().getNewBlockVersion(1047625), BLOCK_VERSION_SC_SUPPORT);
 }
+
+TEST(ForkManager, SidechainVersionForkMainnet) {
+    SelectParams(CBaseChainParams::MAIN);
+
+    // TODO: set proper fork height value.
+    int sidechainVersionForkHeight = 1100000;
```

Comment:
Set higher temporary activation height. Extend test with testnet and regtest coverage.

Reply:
Fixed with commit [190d49622f7e7e70a9d60a74426dc82c813bbb20](https://github.com/HorizenOfficial/zen/pull/435/commits/190d49622f7e7e70a9d60a74426dc82c813bbb20).